### PR TITLE
youtube-dl: wrap with atomicparsley

### PR DESCRIPTION
**Pull request by Profpatsch**
Otherwise --embed-thumbnail will crash youtube-dl while running.
- [x] doesn’t crash anymore

### CODE REVIEW

**Comment by globin** [comment]
Travis failure look relevant

**Comment by Profpatsch** [comment]
Apparently some person thought it was funny to pass `null` into the module. I changed the module to boolean flags.

- [x] Tested `youtube-dl` and `youtube-dl-light` by looking at the wrappers and trying a download

File: pkgs/tools/misc/youtube-dl/default.nix [hunk 1]
```diff
@@ -19,16 +21,20 @@ buildPythonApplication rec {
...
 
-  buildInputs = [ makeWrapper zip pandoc ];
+  buildInputs = [ makeWrapper zip ] ++ optional generateManPage pandoc;
 
   # Ensure ffmpeg is available in $PATH for post-processing & transcoding support.
-  postInstall = stdenv.lib.optionalString (ffmpeg != null)
-    ''wrapProgram $out/bin/youtube-dl --prefix PATH : "${ffmpeg.bin}/bin"'';
+  # atomicparsley for embedding thumbnails
+  postInstall = let
+    ps = [ atomicparsley ] ++ optional ffmpegSupport ffmpeg;
```
Comment (globin) [inline pkgs/tools/misc/youtube-dl/default.nix]: A more descriptive name would be appreciated :)

Comment (Profpatsch) [inline pkgs/tools/misc/youtube-dl/default.nix]: for a variable used once in the line below?

Comment (Profpatsch) [inline pkgs/tools/misc/youtube-dl/default.nix]: done.

**Pull request merged**